script.module.pyxbmct/changelog.txt

```diff
@@ -0,0 +1,20 @@
+[B]Version 1.3.0[/B]
```

Comment:
Please exclude the changelog.txt from the submission and use the `<news>` tag in addon.xml for the changes in the latest version.